### PR TITLE
Created header and main

### PR DESCRIPTION
**Pull request by LazarevSlava**
# Changed

- created Header and Main components
- installed sass
- created vars according design 
- 

## Checklist

- [x] Linter checks
- [x] Code style checks

### CODE REVIEW

**Comment by LazarevSlava** [comment]
I really  need help

fucking lint. If I fix it app doesn’t  work, if app work lint isn’t  satisfied

File: .stylelintrc.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
-  "extends": ["stylelint-config-standard"],
+  "extends": ["stylelint-config-standard", "stylelint-config-recommended-scss"],
```
Comment (michey85) [inline .stylelintrc.json]: Man, we discussed to not use SCSS this time. Only CSS-modules.

File: src/components/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import style from './Header.module.scss';
+
+function Header() {
+  return <div className={style.head}></div>;
```
Comment (michey85) [inline src/components/Header.jsx]: If an element is empty it has to be self-closed.

File: src/components/Header.module.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+@import './utils/index';
+
+.head {
+  width: 100%;
+  font-family: $HomeFont;
+  height: $header-size;
+  background: $darkCyan;
+  background-image: url('../assets/images/bg-header-mobile.svg?react');
```
Comment (michey85) [inline src/components/Header.module.scss]: Are you sure you need `?react` here? It isn't a component, just a path.

File: src/components/Main.jsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import style from './Main.module.scss';
+
+function Main() {
+  return <div className={style.back}></div>;
```
Comment (michey85) [inline src/components/Main.jsx]: If an element is empty it has to be self-closed.

File: src/components/utils/_vars.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
+// colors
+$darkCyan: hsl(180deg, 29%, 50%);
+$lightGrayishCyanBackground: hsl(180deg, 52%, 96%);
+$lightGrayishCyanFilterTablets: hsl(180deg, 31%, 95%);
+$darkGrayishCyan: hsl(180deg, 8%, 52%);
+$veryDarkGrayishCyan: hsl(180deg, 14%, 20%);
```
Comment (michey85) [inline src/components/utils/_vars.scss]: Please use CSS custom properties instead of these.

File: src/index.css [hunk 6]
```diff
@@ -1,4 +1,4 @@
-:root {
+/* :root {
```
Comment (michey85) [inline src/index.css]: Why do we need this commented code?

**Review by michey85 — CHANGES_REQUESTED** [review]
Please use CSS-modules as we agreed before and like it's said in the READ.me

File: package.json [hunk 7]
```diff
@@ -26,7 +26,9 @@
     "eslint-plugin-react-refresh": "^0.4.5",
     "prettier": "^3.2.5",
     "stylelint": "^16.2.1",
+    "stylelint-config-recommended-scss": "^14.0.0",
```
Comment (michey85) [inline package.json]: We don't need this dependency

File: package.json [hunk 8]
```diff
@@ -26,7 +26,9 @@
...
     "stylelint-config-standard": "^36.0.0",
+    "stylelint-scss": "^6.2.1",
```
Comment (michey85) [inline package.json]: We don't need this dependency

File: src/index.css [hunk 9]
```diff
@@ -1,71 +1,53 @@
...
-  }
-
-  &:focus,
-  &:focus-visible {
-    outline: 0.25rem auto -webkit-focus-ring-color;
-  }
+.back {
+  background-color: var(--light-grayish-cyan-background);
+  width: 100%;
+  height: calc(100vh - var(--header-size));
```
Comment (michey85) [inline src/index.css]: Why here and not as a component css module?

Comment (LazarevSlava) [reply src/index.css]: I made all styles modules in next branch

File: package.json (hunk 7 above)
Comment (michey85) [reply package.json]: Why did you remove `stylelint`? We don't need anything related to scss, but stylint was here for a purpose.

Comment (LazarevSlava) [reply package.json]: ok

File: package.json [hunk 10]
```diff
@@ -26,6 +26,7 @@
     "eslint-plugin-react-refresh": "^0.4.5",
     "prettier": "^3.2.5",
     "stylelint-config-standard": "^36.0.0",
+    "stylelint-config-recommended-scss": "^14.0.0",
```
Comment (michey85) [inline package.json]: Why?

File: src/components/Main.jsx [hunk 11]
```diff
@@ -1,3 +1,5 @@
+import './Main.css';
```
Comment (michey85) [inline src/components/Main.jsx]: No. We discussed CSS modules https://github.com/css-modules/css-modules

File: package.json (hunk 7 above)
Comment (LazarevSlava) [reply package.json]: you completely confused me

File: src/components/Main.jsx (hunk 11 above)
Comment (LazarevSlava) [reply src/components/Main.jsx]: ок

File: package.json (hunk 10 above)
Comment (LazarevSlava) [reply package.json]: I installed stylelint again

File: package.json [hunk 12]
```diff
@@ -14,7 +14,8 @@
   },
   "dependencies": {
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "stylelint": "^16.3.0"
```
Comment (michey85) [inline package.json]: It has to be in `devDependencies` section, man

File: package.json (hunk 7 above)
Comment (michey85) [reply package.json]: How? We discussed that we don't need scss and anything related to it. But stylelint itself we need (we use it in our lint script).

**Review by michey85 — APPROVED** [review]
Thanks god!

**Pull request merged**